sonic-syseepromd/scripts/syseepromd

```diff
@@ -121,8 +121,7 @@ class DaemonSyseeprom(daemon_base.DaemonBase):
             self.chassis = sonic_platform.platform.Platform().get_chassis()
             self.eeprom = self.chassis.get_eeprom()
         except Exception as e:
-            self.log_warning(
-                "Failed to load platform-specific eeprom from sonic_platform package due to {}".format(repr(e)))
+            self.log_notice("Failed to load eeprom from sonic_platform package due to {}, retrying using plugin method".format(repr(e)))
```

Comment:
I'm OK with changing the wording here as you did, but I prefer to keep this as a warning, because the old plugins are deprecated. We expect all vendors to migrate to using the sonic-platform API package. This warning makes it clear that the platform is not compliant. Maybe we should also add the word "deprecated", e.g., `retrying using deprecated plugin method`

Reply:
i agree. and we should add it to the ignored list so log analyzer will not fail tests because of that.